substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/jdk/JDKInitializationFeature.java

```diff
@@ -75,8 +75,17 @@ public void afterRegistration(AfterRegistrationAccess access) {
         rci.initializeAtBuildTime("sun.nio", "Core JDK classes are initialized at build time");
         if (Platform.includedIn(Platform.WINDOWS.class)) {
             rci.rerunInitialization("sun.nio.ch.PipeImpl", "Contains SecureRandom reference, therefore can't be included in the image heap");
+            rci.initializeAtRunTime("sun.net.www.protocol.http.ntlm.NTLMAuthentication", "Calls NTLMAuthentication.isTrustedSiteAvailable() on Windows");
         }
 
+        rci.initializeAtRunTime("sun.net.PortConfig", "Calls PortConfig.getLower0() and PortConfig.getUpper0()");
+
+        rci.initializeAtRunTime("sun.nio.ch.DevPollArrayWrapper", "Calls IOUtil.fdLimit()");
```

Comment:
I would also group them together with the place where we initialize at build time.

Reply:
The way this file appears to be organized is with `initializeAtBuildTime()` calls grouped, and other calls grouped. I simply tried to keep this choice.